README.md

```diff
@@ -1,12 +1,11 @@
 # Summary
 
-The configuration file and python file needed to read data from a local
-FPGA
+The configuration file and python file needed to read out and display statistics for FPGAs in the system
 
 
 # Prerequisites
 
-- psycopg2
+- Python2 package psycopg2. Install eg. with `python -m pip install psycopg2`
```

Comment:
Put "psycopg2" in quotes or code quotes to make it clear?

Reply:
I made it italic, that good ?

Reply:
ok

Reply:
Actually, sorry for being late to the party, but AFAIK it is now `pip install psycopg2-binary`. Or even preferred use `yum install python-psycopg2`.

Reply:
All of the commands work for me, including python -m pip... 
I am fine with yum too. Both of you ok to switch it to yum in this case ?